pkg/app/piped/executor/waitapproval/waitapproval.go

```diff
@@ -161,3 +195,28 @@ func (e *Executor) getMentionedAccounts(event model.NotificationEventType) ([]st
 
 	return notification.FindSlackAccounts(event), nil
 }
+
+func (e *Executor) validateApproverNum(approver string, num int) (string, bool) {
+	if num <= 1 {
+		e.LogPersister.Infof("Got approval from \"%s\"", approver)
+		return approver, true
+	}
+	as, ok := e.MetadataStore.Stage(e.Stage.Id).Get(approversKey)
+	if !ok {
+		e.LogPersister.Infof("Got approval from \"%s\"", approver)
+		e.LogPersister.Infof("Waiting for other approvers...")
+		return approver, false
+	}
```

Comment:
```suggestion
	as, _ := e.MetadataStore.Stage(e.Stage.Id).Get(approversKey)
```

Reply:
I'm not sure why you wanna delete this part. Could you tell me the reason?

Reply:
@nghialv 

Should I apply this change to this PR?